xcodebuild/build.go

```diff
@@ -30,8 +29,18 @@ xcodebuild -workspace <workspacename> \
 	[<buildaction>]...
 */
 
-// ArchiveCommandModel ...
-type ArchiveCommandModel struct {
+// const ...
+const (
+	Archive BuildCommand = "archive"
+	Build   BuildCommand = "build"
+	Analyze BuildCommand = "analyze"
+)
+
+// BuildCommand can be {archive, build, analyze}
+type BuildCommand string
```

Comment:
- do not mention the possible values in the comment (they are listed in the `const` block)
- i would call this type as: `Command / Action` and the values: `ArchiveAction, BuildAction ...`

xcodebuild -h:

`xcodebuild [-project <projectname>] [[-target <targetname>]...|-alltargets] [-configuration <configurationname>] [-arch <architecture>]... [-sdk [<sdkname>|<sdkpath>]] [-showBuildSettings] [<buildsetting>=<value>]... !!![<buildaction>]!!!...`

-> it is called: build action